src/Test/UnitTests/attributes.jl

```diff
@@ -1,21 +1,21 @@
 """
-    solver_name(model::MOI.ModelLike, config::Config)
+    test_solver_name(model::MOI.ModelLike, config::Config)
 
 Test that the [`MOI.SolverName`](@ref) attribute is implemented for `model`.
 """
-function solver_name(model::MOI.ModelLike, config::Config)
+function test_SolverName(model::MOI.ModelLike, config::Config)
```

Comment:
The name of the function is not terribly consistent with the other function names (the same holds for all attributes). Maybe something like `test_attribute(model, ::MOI.SolverName, config)`, with the same change for `setup_test`? 

It could also be easier for solvers to implement, with a simple loop over attributes that are supported instead of a longer list of function calls.

Reply:
Yeah this is my naming problem. I don't have a good solution.

Specific attributes:
```Julia
test_attribute_SolverName
test_modification_MultirowChange
test_modification_ScalarCoefficientChange_objective
test_modification_ScalarCoefficientChange_constraint
```

Problem classes?
```julia
test_lp_
test_milp_
test_qp_
test_qcp_
test_nlp_
test_soc_
test_conic_
```

Ideally, we need a formulaic way of generating test names:

`test_` + class + feature + unique identifier?

 - `test_lp_TerminationStatus_INFEASIBLE`
 - `test_lp_TerminationStatus_DUAL_INFEASIBLE`
 - `test_lp_PrimalStatus_INFEASIBILITY_CERTIFICATE`
 - `test_milp_integration_knapsack`
 - `test_soc_VectorOfVariables_extra_terms`
 - `test_soc_VectorAffineFunction_empty_row`

You want to be able to say things like 
"run all `test_soc_` problems excluding `INFEAS`"